<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditerCreature</class>
 <widget class="QWidget" name="EditerCreature">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="supprime_classe">
       <property name="text">
        <string>Supprimer la classe actuelle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="select_classe"/>
     </item>
     <item>
      <widget class="QPushButton" name="ajoute_classe">
       <property name="text">
        <string>Ajouter une classe</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="ajoute_spell">
     <property name="text">
      <string>Ajouter un spell</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tab_spells">
     <column>
      <property name="text">
       <string>Spells</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layout_propx">
     <item>
      <widget class="QLabel" name="text_propx">
       <property name="text">
        <string>Proportion x</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="propx"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layout_propy">
     <item>
      <widget class="QLabel" name="text_propy">
       <property name="text">
        <string>Proportion y</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="propy"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
